src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -193,7 +193,7 @@ struct PropertyName : public std::string {
      * @return true if property is mutable
      */
     bool is_mutable() const {
-        return _mutability == PropertyMutability::RW;
+        return _mutability != PropertyMutability::RO;
```

Comment:
I think it should be reverted. It can change the `get_supported_property` in core implementation as `RO` will can be provided to as supported property for read.
@ilya-lavrenov, could you check this?

Reply:
I don't understand your concern. 

In current PR mutable is RW or WO - properties which can be set to plugin. 
`get_supported_properties` extracts properties, which can be set and `WO` is OK with such criteria, is not it?

Reply:
The question was if we can add `WO` to supported properties. 
If yes then this change is OK (is more intuitive what is mutable). 
Previously only `RW` where in supported properties, just ask if that can cause some regression.